### DISCUSSION TITLE
How does this work with different configuration files and plugin installation directories?

### DISCUSSION THREAD

**Discussion by ElPiloto** [MordechaiHadad/bob#42] [Q&A] [answered]
This is a cool tool.  I'm wondering if bob provides an ergonomic way to switch between different configs and plugin installation directories?  Or is it really just very specific to different nvim versions?  Thanks!

**Comment by MordechaiHadad** [comment, marked as answer]
Hello great question!
Bob is only for switching between neovim versions, if you want a config switcher lookup https://github.com/NTBBloodbath/cheovim

**Comment by NonlinearFruit** [comment]
Bob doesn't really touch your config but you can set `NVIM_APPNAME` to change where neovim reads config ([docs](https://neovim.io/doc/user/starting.html#%24NVIM_APPNAME))

> _setting `$NVIM_APPNAME` to "foo" before starting will cause Nvim to look for configuration files in `$XDG_CONFIG_HOME/foo` instead of `$XDG_CONFIG_HOME/nvim`. `$NVIM_APPNAME` must be a name, such as "foo", or a relative path, such as "foo/bar"_

Elijah Manor's fzf config swapper is a super clever usage of this ([video](https://www.youtube.com/watch?v=LkHjJlSgKZY)) ([gist](https://gist.github.com/elijahmanor/b279553c0132bfad7eae23e34ceb593b))

![image](https://github.com/MordechaiHadad/bob/assets/6962125/f5024ca2-2d63-4fa7-a506-669e17afad82)

I loved his method (great for pairing) the only downside was a fixed number of nvim configs, so I tweaked it ([source](https://github.com/NonlinearFruit/dotfiles/blob/88753bf31ed0b1cda0399ad28e1c7d871c624ac6/scripts/nvims)) making it possible to fuzzy select anything in `.config/nvim-*`. This way you can create as many nvim configs as you want, like `.conflg/nvim-alice` and `.config/nvim-brad`. Or as many premade configs as you want, I use a script ([source](https://github.com/NonlinearFruit/dotfiles/blob/88753bf31ed0b1cda0399ad28e1c7d871c624ac6/setups/nvims.sh)) to make these:
- nvim-vanilla (empty folder)
- nvim-kickstarter
- nvim-nvchad
- nvim-astro
- nvim-lazy
- nvim-doom
